package.json

```diff
@@ -30,6 +30,7 @@
     "cli-unit": "bash lighthouse-core/scripts/run-mocha.sh --cli",
     "viewer-unit": "bash lighthouse-core/scripts/run-mocha.sh --viewer",
     "unit": "bash lighthouse-core/scripts/run-mocha.sh --default",
+    "unit-core": "yarn core-unit",
```

Comment:
lol, this is the worst. Just get rid of `core-unit`. I always forget the order on this as well